x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/feature_flag/comparison.spec.ts

```diff
@@ -11,22 +11,27 @@ import { opbeans } from '../../../fixtures/synthtrace/opbeans';
 const start = '2021-10-10T00:00:00.000Z';
 const end = '2021-10-10T00:15:00.000Z';
 describe('Comparison feature flag', () => {
-  before(async () => {
-    await synthtrace.index(
+  before(() => {
+    synthtrace.index(
       opbeans({
         from: new Date(start).getTime(),
         to: new Date(end).getTime(),
       })
     );
   });
 
-  after(async () => {
-    await synthtrace.clean();
+  after(() => {
+    synthtrace.clean();
   });
 
   describe('when comparison feature is enabled', () => {
     beforeEach(() => {
-      cy.loginAsEditorUser();
+      cy.loginAsEditorUser().then(() => {
+        // enables comparison feature on advanced settings
+        cy.updateAdvancedSettings({
+          'observability:enableComparisonByDefault': true,
+        });
+      });
```

Comment:
This test was relying on the advanced setting `observability:enableComparisonByDefault` to be `true`. However, this is not always the case. We should write test so they can run independently of other tests.

cc @cauemarcondes